bumble/gatt.py

```diff
@@ -231,6 +240,19 @@ def get_advertising_data(self) -> Optional[bytes]:
         """
         return None
 
+    def get_included_service_declaration(self) -> Attribute:
```

Comment:
suggest: create an `IncludedServiceDeclaration`, just like `CharacteristicDeclaration`. This class should have a `service` property that points to the service object it describes (like `CharacteristicDeclaration` has a pointer to its `characteristic`.

Reply:
Done